docs/management/index-patterns.asciidoc

```diff
@@ -89,6 +89,11 @@ To query across all {es} clusters that have been configured for {ccs},
 use a standalone wildcard for your cluster name in your index
 pattern: `*:logstash-*`.
 
+Exclusions can be used too.
```

Comment:
What are exclusions used for? Maybe add something like:

You can use exclusions to narrow the number of indices in your search.

Reply:
I've proposed an alternative below.

Reply:
Amended below